com.krioltech.fnmap.yml

```diff
@@ -0,0 +1,41 @@
+app-id: com.krioltech.fnmap
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: fnmap
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=network
+  - --filesystem=home
+  - --filesystem=xdg-documents
```

Comment:
Why is home needed now? xdg-documents with home is redundant

Reply:
removed home privilege.